domain/src/main/java/org/fao/geonet/domain/userfeedback/Citation.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2001-2016 Food and Agriculture Organization of the
+ * United Nations (FAO-UN), United Nations World Food Programme (WFP)
+ * and United Nations Environment Programme (UNEP)
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or (at
+ * your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ *
+ * Contact: Jeroen Ticheler - FAO - Viale delle Terme di Caracalla 2,
+ * Rome - Italy. email: geonetwork@osgeo.org
+ */
+package org.fao.geonet.domain.userfeedback;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+import org.fao.geonet.domain.GeonetEntity;
+
+/**
+ * A reference to an external resource pointed by a User Feedback.
+ */
+@Entity(name = "GUF_Citation")
+@Table(name = "GUF_Citation")
+@SequenceGenerator(name = Citation.ID_SEQ_NAME, initialValue = 100, allocationSize = 1)
+public class Citation extends GeonetEntity implements Serializable {
+
+    /** The Constant ID_SEQ_NAME. */
+    static final String ID_SEQ_NAME = "gufcit_id_seq";
+
+    /** The Constant serialVersionUID. */
+    private static final long serialVersionUID = -5575769991362660985L;
+
+    /** The id. */
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = Citation.ID_SEQ_NAME)
+    private long id;
+
+    /** The title. */
+    @Column
+    private String title;
+
+    /** The date. */
+    @Column
+    private Date date;
```

Comment:
Would have been better to use same encoding for date as other classes ie. ISODate